apps/src/sites/studio/pages/foorm/forms/editorHelpers.js

```diff
@@ -28,7 +54,16 @@ export function populateCodeMirror() {
   }
 
   codeMirror = initializeCodeMirror(codeMirrorArea, 'application/json', {
-    callback: _.debounce(onCodeMirrorChange, 250)
+    callback: _.debounce(onCodeMirrorChange, 250),
+    getAnnotations: (text, options, cm) => {
+      const cmValidation = cm.getHelper(CodeMirror.Pos(0, 0), 'lint')(
```

Comment:
It feels maybe more natural to me to provide the additional linting we want to do as its own argument here, then do the combining of our linting and code mirror linting in initializeCodeMirror.js. Is that not possible/a bad idea?

Reply:
yeah! i agree, i'll do that